phalcon/mvc/modelinterface.zep

```diff
@@ -158,7 +158,7 @@ interface ModelInterface
 	/**
 	 * Create a criteria for a especific model
 	 *
-	 * @param Phalcon\DiInterface dependencyInjector
+	 * @param \Phalcon\DiInterface dependencyInjector
 	 * @return Phalcon\Mvc\Model\CriteriaInterface
```

Comment:
Actually, what about return types as here - didn't they ought to be prefixed with a backslash as well?